po/fr.po

```diff
@@ -1110,19 +1110,21 @@ msgstr "Articles - %s"
 
 #: src/itemlistformaction.cpp:1511
 msgid "yanq"
-msgstr ""
+msgstr "yanq"
 
 #: src/itemlistformaction.cpp:1533
 #, c-format
 msgid ""
 "Overwrite `%s' in `%s'? There are %d more conflicts like this (y:Yes a:Yes "
 "to all n:No q:No to all)"
 msgstr ""
+"Ecraser `%s' sur `%s'? Il y a %d autre conflits similaires (y : Oui a : Oui "
```

Comment:
A bit undecided here, in a way I could replace the 'yanq' with something more representative in french right, but then in some other cases (line 1126) I do not have the element string before, so the response keys would not be consistent.
What would be the good practice for such translations ?

Reply:
I believe the yes/no ("yn") case is translated to oui/non ("on") already on line 719 of the French .po file:
https://github.com/newsboat/newsboat/blob/3e46c6b6bb8fae79104ef7bdd048e9f64482605c/po/fr.po#L716-L719